### PR TITLE
media-fonts/joypixels: new emoji font

### PR DESCRIPTION
**Pull request by epsilon-0**
Package-Manager: Portage-3.0.14, Repoman-3.0.2
Signed-off-by: Aisha Tammy <gentoo@aisha.cc>

### CODE REVIEW

File: media-fonts/joypixels/joypixels-6.5.0.ebuild [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+FONT_SUFFIX="ttf"
+inherit font
+
+DESCRIPTION="joypixels font for better emoji and unicode support"
+HOMEPAGE="https://www.joypixels.com/"
+
```
Comment (vaukai) [inline media-fonts/joypixels/joypixels-6.5.0.ebuild:11]: Remove empty line here and move KEYWORDS under SLOTS, see https://wiki.gentoo.org/wiki/Basic_guide_to_write_Gentoo_Ebuilds#How_to_create_an_ebuild

Comment (epsilon-0) [reply media-fonts/joypixels/joypixels-6.5.0.ebuild]: done

**Review by juippis — COMMENTED** [review]
Please rebase to resolve the merge conflicts.

**Comment by epsilon-0** [comment]
@juippis done!

**Review by juippis — APPROVED** [review]

**Pull request closed without merging**